test/scripts/e2e_subs/rest.sh

```diff
@@ -112,7 +115,34 @@ function test_assets_endpoint {
   call_and_verify "Asset invalid parameter" "/v2/assets/$ASSET_ID?this-should-fail=200" 400 'parameter detected: this-should-fail'
 }
 
+function pprof_test {
```

Comment:
The pprof auth hits a special case which needs the router configured for a proper test. Covering that with this e2e test is simpler than configuring the router in a unit test.